src/api/shippingAPI.ts

```diff
@@ -0,0 +1,98 @@
+import { getCookie } from '@/libs/manageCookie';
+import type { FieldValues } from 'react-hook-form';
+
+const BASE_URL = process.env.NEXT_PUBLIC_KEYDEUK_API_BASE_URL;
+
+export const postAddress = async (payload: FieldValues) => {
+  const token = await getCookie('accessToken');
+
+  if (!token) {
+    return null;
+  }
+
+  try {
+    const res = await fetch(`${BASE_URL}/api/v1/shipping/address`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${token}`,
+      },
+      body: JSON.stringify(payload),
+    });
+
+    const result = await res.json();
+    return result;
+  } catch (error) {
+    throw error;
+  }
+};
+
+export const getAddresses = async () => {
+  const token = await getCookie('accessToken');
+
+  if (!token) {
+    return null;
+  }
+
+  try {
+    const res = await fetch(`${BASE_URL}/api/v1/shipping/address`, {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${token}`,
+      },
+    });
+
+    const data = await res.json();
```

Comment:
data.data를 return 하는 게 사용처에서 더 편리하지 않을까요?

```suggestion
    const { data }= await res.json();
```